### ISSUE TITLE
Screen Reader NVDA is not reading ariaLabel of Commandbar

### ISSUE THREAD

**Issue by sailakshmi1711558** [microsoft/fluentui#13564]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:
1. Open office fabric select Commandbar control
2. On the NVDA screen reader and check whether it is reading arialable of Command bar i.e.
(ariaLabel="Use left and right arrow keys to navigate between commands") in office fabric control

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar

#### Actual behavior:

Screen Reader NVDA is not reading arialabel of Command Bar

#### Expected behavior:
NVDA screen reader should read arialabel of Command Bar

### Documentation describing expected behavior

![image](https://user-images.githubusercontent.com/66770444/84402890-ae381d00-ac22-11ea-9fef-777f0f72c0a0.png)

**Comment by brandonthomas** [comment]
I have a similar bug. If I muck with the DOM a bit and apply the aria-label to the group (items or far items) it is read out. We need a way to apply this but these two elements are black boxed currently.

**Comment by brandonthomas** [comment]
@paulgildea any update on if you guys are taking this?

**Comment by brandonthomas** [comment]
Ping @paulgildea
